vpr/src/place/net_cost_handler.cpp

```diff
@@ -1478,7 +1489,7 @@ double NetCostHandler::get_net_cube_bb_cost_(ClusterNetId net_id, bool use_ts) {
     ncost = (bb.xmax - bb.xmin + 1) * crossing * chanx_place_cost_fac_[bb.ymax][bb.ymin - 1];
```

Comment:
`is_multi_layer` has the same value in all calls. It can be a member varaible initialized in constructor (or one of the member functions it calls). This would save us an integer comparison for each call to this function.